BadgeFramework/badge_gui_bleak.py

```diff
@@ -0,0 +1,363 @@
+import tkinter as tk
```

Comment:
@Era-Dorta remove this file and make badge_gui use `from badge import OpenBadge`

Reply:
Review this file and remove the original gui file